src/elements/Main.jsx

```diff
@@ -0,0 +1,114 @@
+import { Routes, Route } from "react-router-dom";
+import Category from "./Category";
+import ReviewById from "./review_by_id";
+import Home from "./Home";
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+function Main() {
+  const [reviewState, setReviewState] = useState([]);
+  useEffect(() => {
+    axios
+      .get("https://games-review-hosting.onrender.com/api/reviews")
+      .then((response) => {
+        setReviewState(response.data.reviews);
+      });
+  }, []);
+  return (
+    <main>
+      <Routes>
```

Comment:
do your routes need their own component? is there a reason they're not rendered just in your app?